### PR TITLE
New package: tewi-font-2.0.2

### PR DESCRIPTION
**Pull request by asiekierka**
Let's try this again.

### CODE REVIEW

File: srcpkgs/tewi-font/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'tewi-font'
+pkgname=tewi-font
+version=2.0.0
+revision=1
+noarch="yes"
+hostmakedepends="bdftopcf python"
+makedepends="font-util"
```
Comment (Vaelatern) [inline srcpkgs/tewi-font/template]: Why?

File: srcpkgs/tewi-font/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+distfiles="https://github.com/lucy/tewi-font/archive/${version}.tar.gz"
+checksum=cf617626f3b610e39ed40d5737c49afc9dba0cdf77f8cef8b51d75bd040288b0
+
+do_install() {
+	make
+	cd out
+	for f in *.pcf.gz; do
+		vinstall "$f" 644 usr/share/fonts/misc
+	done
+	cd ..
```
Comment (Vaelatern) [inline srcpkgs/tewi-font/template]: Cleaner to do `for f in out/*.pcf.gz; do` than the `cd out` `cd ..`

**Comment by asiekierka** [comment]
Fixed both.

File: srcpkgs/tewi-font/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+font_dirs="/usr/share/fonts/misc"
+short_desc="Programming monospace bitmap font"
+maintainer='Adrian Siekierka <asiekierka@gmail.com>'
+license="MIT"
+homepage="http://github.com/lucy/tewi-font"
+distfiles="https://github.com/lucy/tewi-font/archive/${version}.tar.gz"
+checksum=cf617626f3b610e39ed40d5737c49afc9dba0cdf77f8cef8b51d75bd040288b0
+
+do_install() {
+	make
```
Comment (Vaelatern) [inline srcpkgs/tewi-font/template]: The `make` step should be in a `do_build()` not `do_install()`

**Comment by asiekierka** [comment]
@Vaelatern Apologies for taking so long.

**Pull request closed without merging**